meetings/2023-12-14.md

```diff
@@ -0,0 +1,20 @@
+# ISIS TC Meeting December 14, 2023 @ 1PM MST 
+
+## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
+
+### Attending
+-
+
+## Agenda / Notes
+- PR Size limitations (Kelvin)
+- Should USGS make IAA progress public (Kelvin) 
+- Any PSMRTS follow-ups? (Kris) 
```

Comment:
@KrisBecker I figure I put this in, that discussion post probably wont get much movement after this week so we can tie up loose ends here.